cfgov/unprocessed/apps/teachers-digital-platform/js/survey/grade-level-page.js

```diff
@@ -1,4 +1,4 @@
-const Cookie = require('js-cookie');
+import Cookies from 'js-cookie';
```

Comment:
Using `Cookies` vs `Cookie` is just in following the ES Module example on https://www.npmjs.com/package/js-cookie